test/framework/config.py

```diff
@@ -272,13 +272,14 @@ def test_generaloption_config_file(self):
         sys.path.insert(0, tmpdir)  # prepend to give it preference over possible other installed easyconfigs pkgs
 
         # test with config file passed via environment variable
+        # also test for existence of HOME and USER by adding paths to robot-paths
         installpath_modules = tempfile.mkdtemp(prefix='installpath-modules')
         cfgtxt = '\n'.join([
             '[config]',
             'buildpath = %s' % testpath1,
             'sourcepath = %(DEFAULT_REPOSITORYPATH)s',
             'repositorypath = %(DEFAULT_REPOSITORYPATH)s,somesubdir',
-            'robot-paths=/tmp/foo:%(sourcepath)s:%(DEFAULT_ROBOT_PATHS)s',
+            'robot-paths=/tmp/foo:%(sourcepath)s:%(DEFAULT_ROBOT_PATHS)s:%(HOME)s:/tmp/%(USER)s',
```

Comment:
@brevans this looks good, but there are situation (e.g. my setup) where the test would fail without having `%(DEFAULT_ROBOT_PATHS)s` at the end; that's also why we're not checking for the entire list of `robot_paths`, but just the first *N*...

So, please change to:

```python
'robot-paths=/tmp/foo:%(sourcepath)s:%(HOME)s:/tmp/%(USER)s:%(DEFAULT_ROBOT_PATHS)s',
```